translator/gobgp.py

```diff
@@ -94,13 +100,13 @@ def _build_path(self, ip, event_data={}):
         communities = Any()
         # Standard community
         # Since we pack both into the community string we need to make sure they will both fit
-        if asn < 65536 and community < 65536:
+        if asn < MAX_SMALL_ASN and community < MAX_SMALL_COMM:
```

Comment:
Why variablize this?

Reply:
Same, no "magic values."

Reply:
Realistically, it helps explainability, and we SHOULD document those consts.